### DISCUSSION TITLE
preserving snowflake time travel over DBT runs

### DISCUSSION THREAD

**Discussion by ezerkar** [dbt-labs/dbt-core#9181] [Ideas]
Hi, would like to have the ability to configure a non incremental model to use "INSERT OVERWRITE INTO {{ this }}" instead of "create or replace" so I can still use snowflake time travel even after dbt was run again

**Comment by b-per** [comment]
If you'd like to do this, you could create your own custom materialization. We have [a guide](https://docs.getdbt.com/guides/create-new-materializations?step=1) about how to do it.
